### DISCUSSION TITLE
How you guys organize multiple scenes from the same movie?

### DISCUSSION THREAD

**Discussion by hiddenpants255** [CodyBerenson/PGMA-Modernized#108] [General]
Hi everyone!

I just found out this repo and so exciting to start organizing my library! I have a lots of scenes that comes from the same movie and curious on how should I organize them in Plex and which agent should I use?

Scene from blogging site seems to have unnecessary long title and doesn't looks nice when viewing lots of them at the same time. I prefer to have them groups in the same movie cover. Is that something I can do in Plex?

**Comment by CodyBerenson** [comment]
@hiddenpants255 

Welcome!  We're excited that you can join us.  

Great news!  The functionality you're looking for comes out of the box from Plex, and therefore our agents support it.  

Here's how it works:  let's say you have individual parts that make up the new Raging Stallion release Get a Room Too https://www.gayeroticvideoindex.com/V/5/67325.html

if you properly name your files:

(Raging Stallion Studios) - Get a Room Too (2021) Scene 1.mp4
(Raging Stallion Studios) - Get a Room Too (2021) Scene 2.mp4
(Raging Stallion Studios) - Get a Room Too (2021) Scene 3.mp4
(Raging Stallion Studios) - Get a Room Too (2021) Scene 4.mp4

and then match using our GEVI agent, Plex will group all four entries into one movie, with one poster.   When you click on "Get Info" for that "movie", it will display the individual parts that were grouped to form that movie.  

A couple things to note:
1. For the most part, playback is as if it were one large movie and not multiple scenes,  However, not all playback devices/apps support the playback of stacked media
2. All parts must be of the same file format (e.g. all MP4 or all MKV)
3. All parts should have identical audio and subtitle streams in the same order
4. Plex has a limitation to the # of scenes it will allow you to stack (per their documentation, up to 8; perhaps somewhat larger though....approximately 8 - 10);  Plex recommends using software to join the scenes into one file rather than stacking them if you're facing a large number of scenes for a movie
5. There is also Plex functionality that allows you to "merge" mutiple scenes into one movie/one poster as a fallback
6. “Other Videos” libraries or those using the “Plex Video Files Scanner” do not support stacked content.

Hope this helps.  

Cheers!

Dakota

**Comment by hiddenpants255** [comment]
Good to hear that Plex support multiple parts video!

I tried renaming to let Plex auto-merge parts into a movie and it's working but I can't find anywhere to select specific part to watch.

Also going this way mean I'm losing the ability to track which scene has who staring
correct?

Anyway thanks for the great tool. You're awesome! I hope I can contribute someday.

**Comment by CodyBerenson** [comment]
Hi again @hiddenpants255 

After thinking about your question yesterday, I decided to add a section to our Guide.  Please see below.  

To answer your questions:
1.  but I can't find anywhere to select specific part to watch.  Plex will treat all the individual media as a movie.  You will need to navigate forward or backward to a specific time in the movie to see the individual scenes.
2.  Also going this way mean I'm losing the ability to track which scene has who staring
correct?  Many of the film agents provide individual scene breakdowns, with description and actors.  See the example I provided in the guide.  Not all films have indivdual scene breakdowns.  

There is an alternative way to approach this that may or may not work better for you.  It will preserve the ability to play each scene separately, yet combine them in plex under one poster.  I'll work on documenting this next.

**Comment by CodyBerenson** [comment]
@hiddenpants255 
Here is a 2nd method for handling scenes where each individual scene is separated and you can maintain your naming convention but are combined under one poster.  For this example, I used Raging Stallion's Get a Room and its 6 individual scenes.  Again, this is native functionality to Plex which works with our agents. 

Step 1.  Create a Parent folder for the movie.  I used the file naming convention of  (Raging Stallion Studios) - Get a Room (2020)
Step 2.  Within that folder, create a child folder called Scenes
Step 3.  In the parent folder, put Scene 1 properly named so that the GEVI agent is able to match the movie.  I used (Raging Stallion Studios) - Get a Room (2020) Scene 1 Dante Colle & Ricky Larkin.mp4

![image](https://user-images.githubusercontent.com/40373801/127359669-36f28a3f-df3d-4851-beaf-0c7929fe357c.png)

Step 4.  In the Scenes folder, put the other 5 scenes named however you want to name them.  

![image](https://user-images.githubusercontent.com/40373801/127359937-2e2f7ee6-734d-428a-99f7-fca53491008c.png)

Once plex matches, it will return one media item with one poster.  When you click on the poster, you will get the full description from the GEVI site for this movie. Scene 1 will play if you click on the poster.  Unlike in the earlier example Plex doesn't "stack" the scenes into one "film" and treats them individually, so you will now also have scenes 2 - 6 displayed individually below.  

![image](https://user-images.githubusercontent.com/40373801/127360764-8559cdc0-f21b-40cf-a684-85f32b5a01b2.png)

Hope this helps.

**Comment by CodyBerenson** [comment]
Here is the update that will be made to the usage guide that incorporates both methods, if it helps.  I encourage feedback.  

[Saving multiple files as a movie.pdf](https://github.com/CodyBerenson/PGMA-Modernized/files/6895098/Saving.multiple.files.as.a.movie.pdf)

**Comment by hiddenpants255** [comment]
The second approach looks very promising. This way I can keep the actors in the filename and still make sense when browsing without Plex.

**Comment by JPH71** [comment]
Apart from using the word *Scene* Plex also recognises the following: cd,
disc, disk, dvd, part and *pt* to create stacked movies
Could you also add this to the pdf?

Brilliant find on showing the scenes as Extras!!!!!! I am so going to start
using that!

Jason

**Reply by CodyBerenson** [reply]
Guide has been updated.
